integration-tests/src/test/java/tachyon/shell/TfsShellTest.java

```diff
@@ -1104,7 +1104,10 @@ public void copyToLocalWildcardHierTest() throws IOException, TachyonException {
    * Checks whether the given file is actually persisted by freeing it, then
    * reading it and comparing it against the expected byte array.
    *
-   * @param file
+   * @param file The file to persist
+   * @param size The size of the file
+   * @throws TachyonException Thrown when the file cannot be freed
+   * @throws IOException Thrown either when the file cannot be freed or read
```

Comment:
This seems to indicate when the file cannot be freed either of the two can be thrown? The current distinction between TachyonException and IOException is if the exception is thrown by a call within Tachyon a TachyonException is thrown. If an exception is thrown by the underlying storage/IO attempt, an IOException will be thrown.